server/lib/nl/constants.py

```diff
@@ -186,8 +181,19 @@
     'sf east bay',
     # US
     'united states',  # need this because the word "states" gets replaced.
+    'usa',
 ])
 
+# Replace the detected place text with this alternate (shorter) place text.
+# The replacement text (value) should be contained within the detected text (key).
+# This ensures that the place string removal from queries will still remove valid place strings.
```

Comment:
I don't understand this constraint.  Maybe an example?

Reply:
Added more comment text